lib/sles4sap_publiccloud.pm

```diff
@@ -211,8 +214,99 @@ sub stop_hana {
         return ();
     }
     else {
-        $self->run_cmd(cmd => $cmd, runas => get_var("SAP_SIDADM"), timeout => $timeout);
+        $self->run_cmd(cmd => $cmd, runas => get_required_var("SAP_SIDADM"), timeout => $timeout);
+    }
+}
+
+=head2 start_hana
```

Comment:
Who is calling this new function?

Reply:
Yep you are right. This one is not used anymore since "crm resource cleanup" starts the hana instance.